src/main/java/minicraft/screen/ItemListMenu.java

```diff
@@ -15,10 +15,11 @@ static Builder getBuilder(RelPos entryPos) {
 			.setSearcherBar(true);
 	}
 
-	protected ItemListMenu(Builder b, ItemEntry[] entries, String title) {
+	protected ItemListMenu(Builder b, ItemEntry[] entries, String title) { this(b, entries, title, true); }
+	protected ItemListMenu(Builder b, ItemEntry[] entries, String title, boolean localizeTitle) {
 		super(b
 			.setEntries(entries)
-			.setTitle(title)
+			.setTitle(title, localizeTitle)
```

Comment:
This looks like you just did a find and replace. Dig a little deeper and try to actually see what this breaks.

Reply:
This works fine.

Reply:
Just remove all the GUI related stuff. It should be its own pull request. (not saying you should open yet another pull request)

Reply:
This pull request completes and resolves duplicated localization requests.

Reply:
Just remove it.

Reply:
Hmm? Isn't it resolving that?